general/cuda.cpp

```diff
@@ -30,23 +34,44 @@ void mfem_cuda_error(cudaError_t err, const char *expr, const char *func,
 void* CuMemAlloc(void** dptr, size_t bytes)
 {
 #ifdef MFEM_USE_CUDA
+#ifdef MFEM_TRACK_CUDA_MEM
+   mfem::out << "CuMemAlloc(): allocating " << bytes << " bytes ... "
```

Comment:
Needs a newline

Reply:
This is intentional: the line is completed with `done` after the cuda call. If the cuda call fails, the error function will print 2 newlines before it prints the error message.